service/store/store.go

```diff
@@ -688,14 +690,21 @@ func (s *Store) GC(discardOrphanDealsAfter time.Duration) (bidsRemoved, filesRem
 		if info.IsDir() {
 			return nil
 		}
-		if _, exists := keepFiles[path]; !exists {
-			if err := os.Remove(path); err != nil {
-				log.Errorf("error removing %s: %v", path, err)
-				return nil
+		shouldRemove := false
+		if _, exists := removeFiles[path]; exists {
+			shouldRemove = true
+		} else if _, exists := keepFiles[path]; !exists {
+			if time.Since(info.ModTime()) > discardOrphanDealsAfter {
+				shouldRemove = true
```

Comment:
Slight improvement. Currently, all files without valid status are removed when running GC. This may include the files downloaded manually but weren't imported to lotus yet. With this change, files for `BidStatusFinalized` are removed immediately to clean up space, but other files are treated as orphans and only removed after 24 hours.